src/components/layout/aside/aside.component.scss

```diff
@@ -2,8 +2,7 @@
 
 // prettier-ignore
 :host {
-  ---wcp-aside-collapsed-width: var(--wcp-aside-collapsed-width, #{utils.size(5)});
-  ---wcp-aside-expanded-width: var(--wcp-aside-expanded-width, #{utils.size(24)});
+  ---wcp-aside-max-width: var(--wcp-aside-max-width, #{utils.size(24)});
```

Comment:
All custom properties have three dashes ('---') at the start not just two.
Is there a reason for that?
I have missed that on the other elements.

Reply:
This should be this way in _every_ component! The idea is to allow theming in a convenient place somewhere up the DOM (e.g. `:root`) and thus, inheriting **public props** (_two dashes!_) in every component. So the usage inside the component is mapped (with defaults) to the **internal props** (_triple dashes!_).

Reply:
Ah ok, didn't know that 👍